src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -13,6 +13,16 @@
     </ItemGroup>
   </Target>
 
+  <!--
+    EmbeddedFiles are source files to be embedded to the PDB. 
+    We need to set Visible to false in order to not display duplicate entries in project UI.
```

Comment:
These are build only files. Don't think we should be putting solution explorer UI changes here.

Reply:
@davkean  Recommended this approach. This metadata property needs to be set in some targets file for the design build to see it.
@davkean, would it be better to add it to MSBuild\Microsoft\VisualStudio\Managed\Microsoft.CSharp.DesignTime.targets?

Reply:
We don't have a great split here - but I can live adding it to CSharp.DesignTime.targets. Does VB use this item?

Reply:
Yes. All managed compilers do including VB and F#.

Reply:
In theory, C++ might join the club too.

Reply:
Okay at it to the Managed.targets if C++ joins the club they'll need to add it themselves.

Reply:
Yes, i think that's fine. unlikely to happen any time soon. I'll move this change to project-system repo then. Thanks!